semgrep_output_v1.atd

```diff
@@ -1113,29 +1114,35 @@ type scan_request = {
 }
 
 (* Response by the backend to the CLI to the POST /scans *)
-(* TODO
 type scan_response = {
-    scan: scan_response_bis;
+    scan: scan_info;
+    scan_config: scan_configuration;
```

Comment:
super-nit: prefixing the fields with `scan` feels redundant. What do you think about naming them `info` and `config` respectively?

Reply:
made a couple of tweaks.